automation/evidence/generate_cron_workflow_evidence.sh

```
@@ -0,0 +1,256 @@
+python3 -m venv "$VENV_DIR"
+"$VENV_DIR/bin/python" -m pip install -U pip setuptools wheel >/dev/null
+
+INSTALL_MODE="${EVIDENCE_INSTALL_MODE:-editable}"
+if [[ "$INSTALL_MODE" == "pypi" ]]; then
+  AUTOMATION_VERSION="${EVIDENCE_PR_AUTOMATION_VERSION:-0.2.37}"
+  ORCH_VERSION="${EVIDENCE_ORCHESTRATION_VERSION:-0.1.21}"
+  PIP_INSTALL_COMMON=(--no-cache-dir --index-url https://pypi.org/simple)
+  "$VENV_DIR/bin/pip" install "${PIP_INSTALL_COMMON[@]}" "jleechanorg-orchestration==${ORCH_VERSION}" >/dev/null
+  "$VENV_DIR/bin/pip" install "${PIP_INSTALL_COMMON[@]}" "jleechanorg-pr-automation[dev]==${AUTOMATION_VERSION}" >/dev/null
+else
+  "$VENV_DIR/bin/pip" install -e "$AUTOMATION_DIR[dev]" >/dev/null
```

Shellcheck (0.11.0)

[error] Use braces when expanding arrays, e.g. ${array[idx]} (or ${var}[.. to quiet).

(SC1087)